redun/executors/gcp_utils.py

```diff
@@ -74,7 +74,8 @@ def make_volume(bucket):
         runnable.container = batch_v1.Runnable.Container()
         runnable.container.image_uri = image
         runnable.container.commands = commands
-        runnable.container.volumes = [','.join([f"{x.mount_path}:{x.mount_path}" for x in volumes])]
+        if volumes:
+            runnable.container.volumes = [','.join([f"{x.mount_path}:{x.mount_path}" for x in volumes])]
```

Comment:
This is needed to avoid submitting an empty string for the volume which lead to this error:

```
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error reopening /dev/null inside container: open /dev/null: permission denied: unknown.
```